packages/c/catbox/package.yml

```diff
@@ -1,14 +1,16 @@
 name       : catbox
 version    : 1.7.0
-release    : 6
+release    : 7
 source     :
     - https://files.pythonhosted.org/packages/source/c/catbox/catbox-1.7.0.tar.gz : f0f066048679a9f5a00b9ccd83c06fb0b98d5d5f27dab1ba0c6ef325e71db109
+homepage   : https://pypi.org/project/catbox/
 license    : GPL-2.0-or-later
 component  : system.devel
```

Comment:
`system.devel` is not a good component for this, can you figure out a better one?

Reply:
what about `programming.library` ,`programming.devel`, or anything on `programming` ?

Reply:
programming.devel maybe?